.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,56 @@
+name: Run Tests
+on: [push, pull_request]
+
+jobs:
+  tests:
+    name: Run Tests
+    defaults:
+      run:
+        shell: bash -l {0}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    env:
+      OS: ${{ matrix.os }}
+      PYTHON: ${{ matrix.python-version }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set up Python ${{ matrix.python-version }}
+        uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          miniconda-version: "latest"
+          python-version: ${{ matrix.python-version }}
+          environment-file: ci/test_environment.yml
+          activate-environment: linsolve_tests
+
+      - name: Conda Info
+        run: |
+          conda info -a
+          conda list
+          PYVER=`python -c "import sys; print('{:d}.{:d}'.format(sys.version_info.major, sys.version_info.minor))"`
+          if [[ $PYVER != $PYTHON ]]; then
+            exit 1;
+          fi
+
+      - name: Install linsolve
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .[testing]
```

Comment:
It looks like the `setup.py` doesn't have an `extras_require` section, which is what the `[testing]` part would be checking against. I think it can be removed without harming anything (so that the installation is just `pip install -e .`).

Reply:
Hit merge a little too fast (before I saw this), but I'll address it in my next PR.